editor/src/components/inspector/sections/layout-section/conditional-section.tsx

```diff
@@ -294,7 +294,7 @@ export const ConditionalSection = React.memo(({ paths }: { paths: ElementPath[]
   )
 
   const branchLabels = useEditorState(
-    Substores.metadata,
```

Comment:
Is this change still needed?

Reply:
Yes, it is needed, because it calls the labelSelector which relies on projectContents too.